tests/drivers/spi/spi_controller_peripheral/testcase.yaml

```diff
@@ -64,6 +64,17 @@ tests:
       - nrf54h20dk/nrf54h20/cpurad
       - nrf54h20dk/nrf54h20/cpuppr
 
+  drivers.spi.spis_fast:
+    # SPIS120 instance occupies P6 which is not available on nRF54H20 DK pin headers
+    # It is added only to check the compilation.
+    build_only: true
+    extra_args: DTC_OVERLAY_FILE="boards/nrf54h20dk_nrf54h20_cpuapp_fast_spis.overlay"
+    platform_exclude:
+      - nrf52840dk/nrf52840
```

Comment:
nrf52840 does not equipped with high speed SPIS

Reply:
it's `platform_exclude`. This approach must be taken because in `common` section there is platform allow and we cannot overwrite it. We can only exclude some of them.